generators/app/index.js

```diff
@@ -1,15 +1,23 @@
 /* eslint-disable consistent-return */
 const chalk = require('chalk');
+
 const AppGenerator = require('generator-jhipster/generators/app');
+
+const constants = require('generator-jhipster/lib/constants/priorities.cjs');
```

Comment:
correct the indentation (eslint can help you) install it with 
```
npm install eslint --global 
```
after that you can run eslint --fix .